eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -147,31 +149,102 @@ private void build(final File project) throws IOException {
                 target
             );
         }
-        Logger.info(this, "Building rust project..");
-        try (
-            VerboseProcess proc = new VerboseProcess(
-                new ProcessBuilder("cargo", "build")
-                    .directory(project)
-            )
-        ) {
-            proc.stdout();
-        } catch (final IllegalArgumentException exc) {
-            throw new BuildFailureException(
+        if (BinarizeMojo.sameProject(
+            project.toPath(), this.cache
+            .resolve("Lib")
+            .resolve(project.getName())
+        )) {
+            Logger.info(
+                this,
+                String.format(
+                    "content of %s was not changed since the last launch",
+                    project.getName()
+                )
+            );
+        } else {
+            Logger.info(
+                this,
                 String.format(
-                    "Failed to build cargo project with dest = %s",
-                    project
-                ),
-                exc
+                    "Building %s rust project..",
```

Comment:
@levBagryansky Double dots? IT is a typo or you put some meaning here?

Reply:
@Graur Yes. This is a long process with the following comments then.